### PR TITLE
JP-178: fix: now dateCounter should work properly

### PR DESCRIPTION
**Pull request by Ixpiria**

### CODE REVIEW

File: frontend/pages/journey/[id].vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script setup lang="ts">
 import { T, useTranslate } from "@tolgee/vue";
-import { differenceInDays, format } from "date-fns";
+import { differenceInDays, differenceInHours, format } from "date-fns";
 import JSConfetti from "js-confetti";
 import QRCode from "qrcode";
 import resolveConfig from "tailwindcss/resolveConfig";
```
Comment (stefll) [inline frontend/pages/journey/[id].vue:6]: date counter does not work properly 
image

**Review by stefll — CHANGES_REQUESTED** [review]
Date counter does not work

File: frontend/pages/journey/[id].vue [hunk 2]
```diff
@@ -119,19 +119,15 @@ QRCode.toDataURL(journeyData.value.invite, opts, function (error, url) {
 const fromDate = new Date(journeyData.value.from.split("T")[0]);
 const toDate = new Date(journeyData.value.to.split("T")[0]);
 const currentDate = new Date();
-const days = ref(Math.ceil(differenceInDays(fromDate, currentDate) / 24));
+const days = ref(Math.ceil(differenceInHours(fromDate, currentDate) / 24));
 const daystoEnd = ref(Math.ceil(differenceInHours(toDate, currentDate) / 24));
+console.log(daystoEnd.value);
+console.log(days.value);
```
Comment (EinSev) [inline frontend/pages/journey/[id].vue]: please remove console.log

**Review by EinSev — CHANGES_REQUESTED** [review]

**Review by EinSev — APPROVED** [review]

**Review by stefll — APPROVED** [review]
Design Review: Looks good

**Pull request merged**